native/src/main/java/io/ballerina/lib/ibm.ibmmq/MQThreadFactory.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) 2023, WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.ballerina.lib.ibm.ibmmq;
+
+public class MQThreadFactory implements java.util.concurrent.ThreadFactory {
```

Comment:
lets provide a thread-factory name in the constructor.

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerinax-ibm.ibmmq/pull/3/commits/970f23591546d20236444487ebbc6138e7320a21